notification/store.go

```diff
@@ -471,3 +479,26 @@ func scanStatus(row scannable) (*SendResult, time.Time, error) {
 
 	return &s, createdAt.Time, nil
 }
+
+// ServiceInfo will return the name of the given service ID as well as the current number
+// of unacknowledged alerts.
+func (s *Store) ServiceInfo(ctx context.Context, serviceID string) (string, int, error) {
```

Comment:
is this function necessary? it looks like it isn't being used anywhere; the call you added in `engine/sendmessage.go` uses the function in `alert/store.go`

I commented the function out and sent an alert to a webhook, and the alert still included the service name/id just fine

Reply:
ah you're right good catch, this was just leftover code I forgot to remove